src/Test/Utilities/Portable/Compilation/ControlFlowGraphVerifier.cs

```diff
@@ -85,7 +86,7 @@ public static string GetFlowGraph(Compilation compilation, ControlFlowGraph grap
                     stringBuilder.AppendLine();
                 }
 
-                appendLine($"Block[B{i}] - {block.Kind}");
+                appendLine($"Block[B{i}] - {block.Kind}{(block.IsReachable ? "" : " [UnReachable]")}");
```

Comment:
>UnReachable [](start = 85, length = 11)

Nit: capitalization is strange. `Unreachable` would have been fine.

Reply:
> Nit: capitalization is strange. Unreachable would have been fine.

I like it this way. Strange is good here, unreachable blocks are uncommon and attention should be drawn to that in tests.

---
In reply to: [177190246](https://github.com/dotnet/roslyn/pull/25698#discussion_r177190246) [](ancestors = 177190246)